cli/src/deploy/deploy.rs

```diff
@@ -0,0 +1,204 @@
+use hyper::{client::Client, Body, Request};
+use serde::{Deserialize, Serialize};
+use serde_json::{from_reader, from_str, json, Value};
+use spinner::SpinnerBuilder;
+
+use std::collections::HashMap;
+use std::error::Error;
+use std::fmt::Display;
+use std::fs::{File, read};
+use std::io::BufReader;
+use std::path::Path;
+
+use ascii_table::{AsciiTable, Column, Align};
+use base64;
+
+const URL: &str = "https://deploy.alantechnologies.com";
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Debug, Serialize)]
+struct AWSCredentials {
+  accessKeyId: String,
+  secretAccessKey: String,
+}
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Debug, Serialize)]
+struct AWSConfig {
+  credentials: AWSCredentials,
+  region: String,
+  cloudProvider: String,
+}
+
+#[allow(non_snake_case)]
+#[derive(Deserialize, Debug)]
+struct App {
+  id: String,
+  url: String,
+  cloudProvider: String,
+  cloudAlias: String,
+}
+
+const CONFIG_NAME: &str = ".alan/deploy.json";
```

Comment:
This isn't `~/.alan/deploy.json`?

Reply:
atm it is so it works with alan deploy, but it will change for both once we settle on the new configuration flow and schema